src/less/components/DittnavLenkePanel.less

```diff
@@ -49,11 +49,21 @@
       flex-direction: column;
     }
   }
-}
 
-#oversiktspanel__header-lenke-id {
-  border-bottom: 0;
-  &:not(:hover) {
-    text-decoration: none;
+  .lenke-overstyring() {
+    border-bottom: 0;
+    &:not(:hover) {
+      text-decoration: none;
+    }
+    &:focus,
+    &--focus {
+      background-color: @orangeFocus;
```

Comment:
^NAV prøver å gå vekk fra orangeFocus; det "nye" er mørkeblå border rundt lenkepaneler og mørkeblå background med hvit tekst på lenker. Tab litt rundt i design.nav.no for å se :)

Reply:
Ok, endret det til å følge den standarden nå. Bør kanskje endre de andre panelene også på sikt, som fortsatt har orange bakgrunn.